jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -6556,6 +6556,7 @@ private static char convEndian(boolean isBigEndian, char value) {
 	public native <V> V uninitializedDefaultValue(Class<?> clz);
 	public native <V> long valueHeaderSize(Class<V> clz);
 	public native boolean isFlattenedArray(Class<?> clz);
+	private native boolean isFlattenedHelper(Field field);
```

Comment:
I suggest to rename it to `isFieldFlattenedHelper()`

Reply:
addressed in a168ee2